Model/PaymentBooster/FlowManager.php

```diff
@@ -0,0 +1,67 @@
+<?php 
+
+declare(strict_types=1);
+
+namespace Bold\CheckoutPaymentBooster\Model\PaymentBooster;
+
+use Bold\CheckoutPaymentBooster\Model\Http\BoldClient;
+use Magento\Framework\Exception\LocalizedException;
+use Bold\CheckoutPaymentBooster\Model\Config;
+
+class FlowManager {
```

Comment:
Nitpick: the curl brace should be on the next line, per the PSR-12 coding standards.

Reply:
Updated here: ced097ae027a8778c625ca9ac8247af4bd4ff361